index.d.ts

```diff
@@ -1,3 +1,5 @@
+import type { parse } from "es-module-lexer";
```

Comment:
I just realized that this may require `es-module-lexer` to become a non-dev dependency 🤔 If that's the case, then maybe we should just simplify the type down to `[imports: unknown[], exports: unknown[], facade: boolean]`?